### DISCUSSION TITLE
Timed QR Code Availability for Events

### DISCUSSION THREAD

**Discussion by vlltr** [HiEventsDev/Hi.Events#290] [Ideas]
A client might prefer that the QR code is only available on the day of the event or a day prior, especially to prevent issues like resale or scams. For instance, if someone initially can't attend and shares their QR code, but later decides to go, this approach avoids conflicts. This configurable option is particularly beneficial for events planned well in advance.

**Comment by daveearley** [comment]
Hi @vlltr,

There aren't any immediate plans to support this, however, I wouldn't completely rule it out for the future. As a workaround, it would be quite simple to add functionality to rotate a user's ticket. You'd just need to update the `attendees.public_id` value in the DB.
